packages/gestalt/src/ModalAlert/ModalAlertAction.js

```diff
@@ -0,0 +1,50 @@
+// @flow strict
+import { type Node } from 'react';
+import Button from '../Button.js';
+import { type ActionDataType } from '../ModalAlert.js';
```

Comment:
importing this from the parent is a little gross but it does allow us to dedupe types and still use autogen props tables